apis/v1alpha1/cluster_types.go

```diff
@@ -37,8 +37,8 @@ type CrdbClusterSpec struct {
 	Nodes int32 `json:"nodes"`
 	// Container image information
```

Comment:
Seems like we've got a convention here to prefix these comments with `(Optional) ` for optional fields

Reply:
Done